### ISSUE TITLE
`require('esc')` throws "Cannot find module"

### ISSUE THREAD

**Issue by KengoTODA** [pvorb/node-esc#1]
Hi.
I found that `require('esc')` throws "Cannot find module" exception.
This module depends on your [node-append](https://github.com/pvorb/node-append)?
Please edit package.json, republish node-esc and publish node-append. Thank you.

```
node.js:134
        throw e; // process.nextTick error, or 'error' event on first tick
        ^
Error: Cannot find module 'append'
    at Function._resolveFilename (module.js:317:11)
    at Function._load (module.js:262:25)
    at require (module.js:346:19)
    at Object.<anonymous> (/usr/lib/node_modules/<my_module>/node_modules/esc/esc.js:1:76)
    at Module._compile (module.js:402:26)
    at Object..js (module.js:408:10)
    at Module.load (module.js:334:31)
    at Function._load (module.js:293:12)
    at require (module.js:346:19)
```

**Comment by KengoTODA** [comment]
Sorry,

> publish node-append.

The node-append had been published. I cannot find it via search.npmjs.org.

**Comment by pvorb** [comment]
It's not node-append. That's only the name of the github repo. In npm it's [append](http://search.npmjs.org/#/append).

I don't have the time to fix this right now, but I hope I can do it this evening.

**Comment by KengoTODA** [comment]
Thank you.

**Comment by pvorb** [comment]
Fixed 393968bf73.

**Comment by KengoTODA** [comment]
I've confirmed that my module works fine.
Thanks a lot!
